services/npm/npm-type-definitions.service.js

```diff
@@ -26,25 +28,26 @@ module.exports = class NpmTypeDefinitions extends NpmBase {
     ]
   }
 
-  static transform({ devDependencies }) {
-    return {
-      supportedLanguages: [
-        { language: 'TypeScript', range: devDependencies.typescript },
-        { language: 'Flow', range: devDependencies['flow-bin'] },
-      ].filter(({ range }) => range !== undefined),
+  static transform({ devDependencies, types }) {
+    const supportedLanguages = []
+
+    if (types !== undefined || devDependencies.typescript !== undefined) {
```

Comment:
Ah, good idea. I had ruled out using `files` key because [it's optional](https://docs.npmjs.com/files/package.json#files) ([example](https://registry.npmjs.org/link-into/latest)), though we could use it when it's there.